.github/workflows/python-package.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.7, 3.9]
+        python-version: ['3.8', '3.7', '3.9', '3.10']
```

Comment:
Just curious, did we first support 3.8 and then 3.7 ?

Reply:
no, It was 3.7 first, some how the order was not serail in test matrix.